### PR TITLE
MS-13 : remove client

### PR DESCRIPTION
**Pull request by amacazzaga**
Just  a few lines of code to remove clients (from 71, to 80)

### CODE REVIEW

**Comment by nicolasblasco** [comment]
@amacazzaga remember to add Damian as a reviewer

**Comment by luisnannini** [comment]
CAREFUL ! you are trying to merge into Main instead of Develop. Also please add a Reviewer and rename the PR title correctly.

**Review by damianalvarez — CHANGES_REQUESTED** [review]
@amacazzaga can you please, point this branch to `develop` and give some format to the PR title?

File: controllers/clients.js [hunk 1]
```diff
@@ -66,6 +66,20 @@ const getByActivity = (req, res) => {
...
 };
+//CLIENT REMOVE///
+
+const remove = (req, res) => {
+  const client = clients.find((client) => client.id === req.params.id);
+  if (client) {
+    const clientsFilter = clients.filter(
+      (client) => client.id !== req.params.id
+    );
+    const clientRemoved = clientsFilter;
```
Comment (damianalvarez) [inline controllers/clients.js:77]: @amacazzaga I think you need to assign the filtered clients to the original clients array here. And return only the cleints filtered array on the line 78

**Review by damianalvarez — CHANGES_REQUESTED** [review]

File: controllers/clients.js [hunk 2]
```diff
@@ -67,28 +67,15 @@ const getByActivity = (req, res) => {
...
-        res.status(201).json(Admins[adminIndex])
-      }
-    })
-  } else {
-    res.status(404).send("Admin not found")
-  }}*/
+const remove = (req, res) => {
+  const client = clients.find((client) => client.id === req.params.id);
+  if (client) {
+    const clientsFilter = clients.filter((client) => client.id !== req.params.id);
```
Comment (damianalvarez) [inline controllers/clients.js:73]: @amacazzaga can we rename this constant to `filteredClients`?

File: controllers/clients.js [hunk 3]
```diff
@@ -67,28 +67,15 @@ const getByActivity = (req, res) => {
...
+    clients = clientsFilter;
+    res.json({client,clients});
```
Comment (damianalvarez) [inline controllers/clients.js:75]: @amacazzaga, can we only return the removed client in the response?

**Review by damianalvarez — CHANGES_REQUESTED** [review]

**Review by damianalvarez — APPROVED** [review]

File: controllers/clients.js [hunk 4]
```diff
@@ -67,28 +67,14 @@ const getByActivity = (req, res) => {
...
-        res.status(201).json(Admins[adminIndex])
-      }
-    })
-  } else {
-    res.status(404).send("Admin not found")
-  }}*/
+const remove = (req, res) => {
+  const client = clients.find((client) => client.id === req.params.id);
+  if (client) {
+    const filteredClients = clients.filter((client) => client.id !== req.params.id);
```
Comment (damianalvarez) [inline controllers/clients.js:73]: @amacazzaga we need to re-assign the filtered clients to the original client's array here

**Review by damianalvarez — CHANGES_REQUESTED** [review]

**Review by damianalvarez — APPROVED** [review]

**Pull request merged**